worker/Dockerfile

```diff
@@ -8,10 +8,10 @@ RUN GO111MODULE=on go mod download
 ARG version=dev
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -ldflags "-w -extldflags '-static' -X main.VersionTag=${version}" -o /worker cmd/worker/worker.go
 
-FROM alpine:3.16
+FROM alpine:3.15
 
 RUN apk add --no-cache \
-  ffmpeg=5.0.1-r1 \
+  ffmpeg=4.4.1-r2 \
```

Comment:
This can be rolled back

Reply:
Ah, nice. When I tried this yesterday the `ffmpeg` command crashed, so I thought this was the solution. It worked afterwards but might be a coincidence.